packages/transactions/src/builders.ts

```diff
@@ -201,11 +372,10 @@ export async function broadcastRawTransaction(
   }
 
   const text = await response.text();
-  try {
-    return JSON.parse(text) as TxBroadcastResult;
-  } catch (e) {
-    return text;
-  }
+  return {
+    // Replace extra quotes around txid string
+    txid: text.replace(/["]+/g, ''),
```

Comment:
I think there needs to be some validation here. There are times when the response is ok, but the text is not a txid. see https://github.com/blockstack/stacks.js/issues/1054

Reply:
@aulneau Added the validation on txid. Give it final look for approval or let me know if it needs further changes. cc: @zone117x